### PR TITLE
Add 2.3.2 release notes

### PR DESCRIPTION
**Pull request by znicholasbrown**
[Rendered release notes](https://github.com/PrefectHQ/prefect/blob/e007c753349411e2806da0c22536001db2596a1a/RELEASE-NOTES.md)

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,12 +1,30 @@
...
+## 2.3.2
+
+### Enhancements
+* UI displays a message when backend is unreachable - https://github.com/PrefectHQ/prefect/pull/6670
+
+### Fixes
+* Fix issue where parameters weren't updated when a deployment was re-applied by @lennertvandevelde in https://github.com/PrefectHQ/prefect/pull/6668
+* Fix issues with stopping Orion on Windows machines - https://github.com/PrefectHQ/prefect/pull/6672
+* Fix issue with GitHub storage running in non-empty directories - https://github.com/PrefectHQ/prefect/pull/6693
+* New deployments now respect all user inputs - https://github.com/PrefectHQ/prefect/pull/6695
```
[the remark this answers is not in the document]
Comment (znicholasbrown) [reply RELEASE-NOTES.md:12]: How about: 
"Fix issue where some user-supplied values were overridden when creating new deployments"?

**Review by madkinsz — APPROVED** [review]

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,12 +1,30 @@
 # Prefect Release Notes
 
+## 2.3.2
+
+### Enhancements
+* UI displays a message when backend is unreachable - https://github.com/PrefectHQ/prefect/pull/6670
```
Comment (billpalombi) [inline RELEASE-NOTES.md:6]:
```suggestion
* UI displays an error message when backend is unreachable - https://github.com/PrefectHQ/prefect/pull/6670
```

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,12 +1,30 @@
...
+
+### Fixes
+* Fix issue where parameters weren't updated when a deployment was re-applied by @lennertvandevelde in https://github.com/PrefectHQ/prefect/pull/6668
+* Fix issues with stopping Orion on Windows machines - https://github.com/PrefectHQ/prefect/pull/6672
+* Fix issue with GitHub storage running in non-empty directories - https://github.com/PrefectHQ/prefect/pull/6693
+* Fix issue where some user-supplied values were ignored when creating new deployments - https://github.com/PrefectHQ/prefect/pull/6695
+
+### Collections
+* [prefect-fugue](https://prefecthq.github.io/prefect-jupyter/)
```
Comment (billpalombi) [inline RELEASE-NOTES.md:15]:
```suggestion
* Added [prefect-fugue](https://prefecthq.github.io/prefect-jupyter/) 
```

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,12 +1,30 @@
...
+
+### Contributors
+* @lennertvandevelde made their first contribution — [https://github.com/PrefectHQ/prefect/pull/6668](https://github.com/PrefectHQ/prefect/pull/6668)
```
Comment (billpalombi) [inline RELEASE-NOTES.md:18]:
```suggestion
* @lennertvandevelde made their first contribution! — [https://github.com/PrefectHQ/prefect/pull/6668](https://github.com/PrefectHQ/prefect/pull/6668)
```

**Review by billpalombi — APPROVED** [review]
Small tweaks. Otherwise lgtm

**Pull request merged**